openworld/sdk/generator/client/templates/client.jinja2

```diff
@@ -36,7 +36,7 @@ Args:
             header.USER_AGENT: self.__user_agent,
         {% for arguemnt in operation.snake_case_arguments_list %}
             {% if arguemnt.in_.value == 'header' %}
-            '{{ arguemnt.alias }}': json.loads(json.dumps({{ arguemnt.name.strip() }})),
+            '{{ arguemnt.alias }}': {{ arguemnt.name.strip() }},
```

Comment:
We added and removed the `json.loads(json.dumps(...))` multiple times. Can you please make sure if we need to do this or not?

Reply:
Yes, we need it when the header datatype is custom (model, enum, ...etc). Though in any case, serialization and then deserialization of the same object should not do anything wrong in theory, but when it comes to strings, in practice say we serialize the string `"Name"`, deserializing it returns `"`Name`"`, which when sent to the API, is dealt with as malformed string. That is why, now we only do serialization and deserialization only when the header is a model or enum.